floodviz/static/js/hydrograph.js

```diff
@@ -89,13 +117,12 @@
 				.key(function (d) {
 					return d.key;
 				})
-				.entries(graph_data);
+				.entries(data);
 			// Loop through each symbol / key
 			dataNest.forEach(function (d) {
-				var map_site = document.getElementById('map' + d.key);
-				map_site.classList.add('accent');
+				FV.map_figure.addaccent(d.key);
```

Comment:
I think Garritt is taking care of this but should not call FV.map_figure. Or alternatively pass in the list of initial gages to the map so that they are accented.